plugins/utils_operators/slack_operators.py

```diff
@@ -0,0 +1,154 @@
+# slack_operators.py - logic for sending failure and success messages from airflow to slack
```

Comment:
Very neat! Want to see it in action. Out of curiosity, have you tried sending a long message that exceeds the Slack limit? If so, what happens?

I ask because the 'get_last_refreshed_gap` DAG, which calculates packages that are behind the refresh schedule and sends the list to Slack, has a really long message. The nasty code I had earlier for sending Slack messages was nasty, in large part, because it had to take this long string, break it into multiple messages under the limit, and send them to the webhook one at a time.

Wonder what happens in those scenarios with this one.

Reply:
No I havent tried - right now this returns a pretty templated message - it has a header and it has a "number of records affected" message.

I dont know any other use cases for the `SlackOperator` yet, but when I get to a point where I'm experimenting with them I'll let you knwo if I run into a message-size-limit issue